docs/localization.md

```diff
@@ -87,8 +134,8 @@ GlotPress currently does not support pluralization using the [`.stringsdict` fil
 
```

Comment:
```suggestion

This is not an ideal situation, and in the future we're hoping to properly support real pluralization with `.stringdict` files, which takes into account the complexity of different locales having different pluralization rules (sometimes way more complex than the simple singular/plural rule that English has, e.g. like [in Irish](https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html#ga)).

In the meantime, we sadly have to make-do with the simplistic solution of providing two different localized strings and doing the pluralization decision by code, even if that will lead to some inexact pluralization in some locales.
```

Reply:
Yes, good comment. Localization and pluralization are complicated 👍 it's also the case in Lithuanian that we have more than two different endings based if there're 1, 2+, or 10+ subjects. In my experience, good translators usually try to get around it in some creative way although it's not always that easy.